{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -6,136 +6,159 @@
 render model specs with the correct location parameters plugged in.
 
 It will look for the model spec template in "model_spec.in" in the directory
-"../{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification".
-Specify multiple locations in a file called "locations.txt" in the same directory.
-You can create a single location using the "-s" option. Using the single
-location option will override an existing locations file.
+``src/{{cookiecutter.package_name}}/model_specifications``.
+Specify multiple locations in a file called "locations.txt" in the same
+directory. You can create a single location using the "-s" option. Using the
+single location option will override an existing locations file.
 
 The application will look for the model spec and locations files based
 on the python environment that is active and these files don't need
 to be specified if the default names and location are used.
 """
+from collections import namedtuple
```

Comment:
I always try to organize imports in blocks:
```
standard lib imports

library imports

internal package imports
```
and as much as possible keep the blocks sorted alphabetically.  I've found this really helpful when doing refactorings or trying to resolve import time issues.